lib/modules/datasource/maven/util.ts

```diff
@@ -485,6 +485,18 @@ export async function getDependencyInfo(
     }
   }
 
+  const relocation = pomContent.descendantWithPath(
+    'distributionManagement.relocation',
+  );
+  if (relocation) {
+    const newGroup = relocation.valueWithPath('groupId') ?? dependency.group;
+    const newName = relocation.valueWithPath('artifactId') ?? dependency.name;
+    result.replacementName = newGroup + ':' + newName;
```

Comment:
For consistency sake, wouldn't these be relocationGroup and relocationName respectively?